arch/arm/src/tlsr82/chip/b87/boot/cstartup_flash.S

```diff
@@ -0,0 +1,593 @@
+/********************************************************************************************************
```

Comment:
So this file is not licensed with Apache?

Reply:
I have changed the license to Apache.